Proposal.txt

```diff
@@ -0,0 +1,4 @@
+Lloyd Page
+A nation simulator, in which the user must manage the govermental functions and policies, including military type,size of military, economic incetives/policies, social policies, etc through text-based input. To allow for more efficient management, the user will have the capacity to use an algorithm to determine the most efficient sorting algorithm given the input data, and how it is sorted. User data will be stored in a .txt file.
+Functions will be provided by sorting algorithms, along with conditional statements, and loops. Structures will be used to keep nation data in a single data type, with a dynamic data structure(probably a linked-list) to store items inside the structure efficiently, and without worrying about size. Strings will be used to santize inputs, and for menu selection/input
+I have done many simulations/games, however this is a genre I am yet to cover, so to have a fresh experience, I chose to do a nation simulator. I also wanted to do something somewhat practical, so creating an optimizer for sorting algorithms is something that I can use further down the road without many problems.
```

Comment:
This is an interesting take on a simulator, and I like that you're using your knowledge of sorting algorithms.  Go for it.